integration-tests/actions/vrf/vrfv2plus/setup_steps.go

```diff
@@ -235,51 +232,100 @@ func setupVRFNode(contracts *vrfcommon.VRFContracts, chainID *big.Int, config *v
 	nodeConfig := node.NewConfig(vrfNode.CLNode.NodeConfig,
 		node.WithKeySpecificMaxGasPrice(vrfNode.TXKeyAddressStrings, *config.CLNodeMaxGasPriceGWei),
 	)
-	l.Info().Msg("Restarting Node with new sending key PriceMax configuration")
+	l.Info().
+		Strs("Sending Keys", vrfNode.TXKeyAddressStrings).
+		Int64("Price Max Setting", *config.CLNodeMaxGasPriceGWei).
+		Msg("Restarting Node with new sending key PriceMax configuration")
 	err = vrfNode.CLNode.Restart(nodeConfig)
 	if err != nil {
 		return fmt.Errorf(vrfcommon.ErrGenericFormat, vrfcommon.ErrRestartCLNode, err)
 	}
 	return nil
 }
 
-func SetupVRFV2PlusWrapperEnvironment(
+func SetupVRFV2PlusWrapperForExistingEnv(
 	ctx context.Context,
+	sethClient *seth.Client,
+	vrfContracts *vrfcommon.VRFContracts,
+	vrfv2PlusTestConfig types.VRFv2PlusTestConfig,
+	consumerContractsAmount int,
 	l zerolog.Logger,
+) (*VRFV2PlusWrapperContracts, *big.Int, error) {
+	config := *vrfv2PlusTestConfig.GetVRFv2PlusConfig()
+	var wrapper contracts.VRFV2PlusWrapper
+	var err error
+	if *config.ExistingEnvConfig.UseExistingWrapper {
+		wrapper, err = contracts.LoadVRFV2PlusWrapper(sethClient, *config.ExistingEnvConfig.WrapperAddress)
+		if err != nil {
+			return nil, nil, fmt.Errorf(vrfcommon.ErrGenericFormat, "error loading VRFV2PlusWrapper", err)
+		}
+	} else {
+		wrapperSubId, err := CreateSubAndFindSubID(ctx, sethClient, vrfContracts.CoordinatorV2Plus)
+		if err != nil {
+			return nil, nil, err
+		}
+		wrapper, err = contracts.DeployVRFV2PlusWrapper(sethClient, vrfContracts.LinkToken.Address(), vrfContracts.MockETHLINKFeed.Address(), vrfContracts.CoordinatorV2Plus.Address(), wrapperSubId)
+		if err != nil {
+			return nil, nil, fmt.Errorf(vrfcommon.ErrGenericFormat, ErrDeployWrapper, err)
+		}
+	}
+	wrapperSubID, err := wrapper.GetSubID(ctx)
+	if err != nil {
+		return nil, nil, fmt.Errorf(vrfcommon.ErrGenericFormat, "error getting subID", err)
+	}
+	wrapperConsumers, err := DeployVRFV2PlusWrapperConsumers(sethClient, wrapper, consumerContractsAmount)
```

Comment:
If suppose a `wrapperConsumerAddress` is defined, then why are we still deploying a new wrapper consumer